### ISSUE TITLE
Exante - AI/ML Engineer

### ISSUE THREAD

**Issue by sprince0031** [sprince0031/GetEmployed#7]
# Job description

## About the Role

Are you passionate about leveraging AI to transform operations in the brokerage industry? Do you thrive in dynamic environments where innovation and flexibility are paramount? Join us as an AI/ML Engineer to spearhead strategic initiatives, enhance our operations, and drive the future of brokerage technology.

**Reporting line**: Head of Client Services

**Location**: anywhere

**Worksite**: flexible (remote/onsite)

## Responsibilities

- Assess the existing IT backlog and companywide initiatives to identify opportunities to integrate AI-driven solutions. Work closely with the IT team to understand existing systems and propose AI-driven alternatives.
- Develop strategies to integrate AI/ML solutions into our trading platforms and internal operations without compromising security.
- Utilize machine learning to alleviate pressure and streamline processes within counterparty onboarding, order categorization, client experience, client data metrics, etc.
- Develop and implement algorithms for data cleansing, preprocessing, and feature engineering.
- Utilize machine learning, statistical analysis, and predictive modeling to extract valuable insights from large datasets.
- Monitor and evaluate the performance of AI/ML models and algorithms in production and continuously iterate to improve their accuracy, efficiency, and scalability.

## Requirements

- Strong programming skills in languages such as Python, Java, or C++.
- Strong understanding of machine learning techniques and frameworks.
- Experience with data preprocessing, feature engineering, and model evaluation.
- Familiarity with deep learning frameworks like PyTorch or TensorFlow.
- Experience with RAG models (embedding, reranking) is a plus.
- Familiarity with developing custom AI models and algorithms.
---
[Job link](https://www.glassdoor.ie/Job/exante-jobs-SRCH_KO0,6.htm)
